### PR TITLE
LPS-84119 Add YMLLongLinesCheck

### PR DESCRIPTION
**Pull request by ling-alan-huang**
Hi Hugo,

**Issue:** https://github.com/brianchandotcom/liferay-portal/pull/79231

**My changes:**
1. I created YMLLongLinesCheck, set 120 to max line length for yaml, I think 80 is too short for yaml.
2. About my commit ["Skip the temporary lines...."](https://github.com/hhuijser/liferay-portal/commit/5d21a719efe101e44aa1aa4c1ea2c327e078fc46),
Before `format()` in [YMLSourceProcessor](https://github.com/hhuijser/liferay-portal/blob/5d21a719efe101e44aa1aa4c1ea2c327e078fc46/modules/util/source-formatter/src/main/java/com/liferay/source/formatter/YMLSourceProcessor.java#L55-L62), there is `_preProcess()` to do some temporary changes for yaml, in order to easily do other `YML * Check`, when all `YML * Check` are finished, `_postProcess()` will revert what `_preProcess()` did.

Basically, `_preProcess()` will change:
```
            parameters:
                - in: path
                  name: userAccountId
                  required: true
```
to
```
            parameters:
                - 
                  in: path
                  name: userAccountId
                  required: true

```

### CODE REVIEW

**Comment by ling-alan-huang** [comment]
ci:test:sf

**Comment by ling-alan-huang** [comment]
ci:test:relevant

**Comment by ling-alan-huang** [comment]
SF failures are not related to my changes.

**Comment by ling-alan-huang** [comment]
The failures are not related to my changes.

**Comment by ling-alan-huang** [comment]
Sent to Brian for review: https://github.com/brianchandotcom/liferay-portal/pull/79414
